### ISSUE TITLE
Display additional markup taken from SGF data

### ISSUE THREAD

**Issue by herzbube** [herzbube/littlego#349]
With the intregration of SgfcKit the app has now access to markup properties that are stored inside an SGF file. The app should be able to display this markup.

**Comment by herzbube** [comment]
SGF defines the following markup properties:

- CR/SQ/TR/MA
  - Data type for all of these is List of Point.
  - A point can only appear in one of these properties.
  - Draw a circle, square, triangle or an "X" on the point(s).
- AR/LN
  - Data type for all of these is List of composed Point : Point.
  - The same pair of points can appear only once per property. SGF does not prohibit the same pair of points from appearing in both AR and LN.
  - Draw an arror and/or a line from the first to the second point.
- LB
   - Data type is List of composed Point : SimpleText
   - Draw a label on the point(s).
- SL
   - Data type is List of Point
   - Markup the point(s) as "selected".
   - Idea 1: Draw a ring around the point. If there is a stone on the point, draw the stone smaller.
   - Idea 2: Use yet another symbol, e.g. a star.
- DD
  - Data type is Elist of Point
  - Dim/grey out the point(s)

Notes:

- Markup is only drawn if the board position with those properties is shown.
- Exception is DD, because that is an inheritable property.

**Comment by herzbube** [comment]
When adding the new markup, also consider whether the rectangle symbol can be improved. I received this user feedback via email:

> A shade of white quadrilateral on a black stone is light and difficult to recognize.

**Comment by herzbube** [comment]
Implementation notes to be added to the in-game manual
- Markup is displayed only in play mode (`UIAreaPlayModePlay`), i.e. not in scoring mode and not in setup mode
- Order of precedence in which markup is drawn
  - A move number takes precedence
    - If a mover number is displayed for the last move it is displayed in red color, compensating for the last move symbol not being displayed
  - Next comes a symbol or label
    - A user preference lets the user decide whether symbols or labels should take precedence
    - The default is symbols
  - Next move marker and last move symbol have lowest priority
    - Reason for low priority: Next move marker and last move symbol are merely a convenience, whereas symbols or labels may be quite important for the understanding of the information in a game node.
- Labels
  - Labels are not displayed if the intersections are too small (same mechanism as with move numbers)
  - A user preference lets the user decide whether labels that are too long should be truncated

When markup editing mode is implemented later on
- In markup editing mode the user can place only either a symbol or a label on the same intersection, so there can be no conflict in game data manufactured by Little Go. But external SGF files might contain both a symbol and a label on the same intersection.
- In markup editing mode no move numbers are displayed, so that symbols and labels cannot be hidden by move numbers
- In markup editing mode the next move marker and last move symbol are also not displayed, so as not to disturb markup editing.

**Comment by herzbube** [comment]
In-game manual updated in commit 22e04d7c04cafa81acbeb882e59bfb367e3cbadf.

**Comment by herzbube** [comment]
Ready for next release, closing this issue.
